g5k_discovery/templates/g5k_discovery/node_details.html

```diff
@@ -1,214 +1,184 @@
+                        <i class="fa fa-caret-right"></i> Chassis
+                    </h4>
+                    <div id="chassis" class="collapse capability-group-body">
+                        <dl class="dl-horizontal">
+                            <dt>Manufacturer</dt><dd>{{ node.chassis.manufacturer }}</dd>
+                            <dt>Name</dt><dd>{{ node.chassis.name }}</dd>
+                            <dt>Serial</dt><dd>{{ node.chassis.serial }}</dd>
+                        </dl>
+                    </div>
                 </div>
-                <div class="col col-sm-3">
-                    <strong>Vendor: </strong> {{ sd.vendor }}
+
+                <div class="capability-group">
+                    <h4 class="capability-group-header" data-toggle="collapse" data-target="#network">
+                        <i class="fa fa-caret-right"></i> Network Adapters
+                    </h4>
+                    <div class="collapse capability-group-body" id="network">
+                        {% for na in node.network_adapters %}
+                        <h5 class="bg-info"><strong>Network Adapter #{{ forloop.counter0 }}</strong></h5>
+                        <dl class="dl-horizontal">
+                            <dt>Bridged</dt><dd>{% if na.bridged == False %}No{% else %}Yes{% endif %}</dd>
+                            <dt>Device</dt><dd>{{ na.device }}</dd>
+                            <dt>Driver</dt><dd>{{ na.driver }}</dd>
+                            <dt>Interface</dt><dd>{{ na.interface }}</dd>
+                            <dt>Mac</dt><dd>{{ na.mac }}</dd>
+                            <dt>Management</dt><dd>{% if na.management == False %}No{% else %}Yes{% endif %}</dd>
+                            <dt>Model</dt><dd>{{ na.model }}</dd>
+                            <dt>Mounted</dt><dd>{% if na.mounted == False %}No{% else %}Yes{% endif %}</dd>
+                            <dt>Rate</dt><dd>{{ na.rate }}</dd>
+                            <dt>Vendor</dt><dd>{{ na.vendor }}</dd>
+                        </dl>
+                        {% endfor %}
+                    </div>
                 </div>
+                <div class="capability-group">
+                    <h4 class="capability-group-header" data-toggle="collapse" data-target="#jobs">
+                        <i class="fa fa-caret-right"></i> Supported Job Types
+                    </h4>
+                    <div id="jobs" class="collapse capability-group-body">
+                        <dl class="dl-horizontal">
+                            <dt>Best Effort</dt><dd>{% if node.supported_job_types.besteffort %}Yes{% else %}No{% endif%}</dd>
+                            <dt>Deploy</dt><dd>{% if node.supported_job_types.deploy %}Yes{% else %}No{% endif%}</dd>
+                            <dt>Virtual</dt><dd>{{ node.supported_job_types.virtual }}</dd>
+                        </dl>
+                    </div>
```

Comment:
I think "job type" is also unused in chameleon, although I guess we might eventually want to add (and distinguish) containers / vm / baremetal somewhere

Reply:
I'm good either way, we can drop it if we want or leave it and eventually repurpose it.